examples/06-verif-manual/vm-002-beam_stresses_and_deflections.py

```diff
@@ -0,0 +1,181 @@
+r"""
+.. _ref_vm5_example:
+Beam Stresses and Deflections
+------------------------------------------------
+
+Problem Description:
+-   A standard 30" WF beam, with a cross-sectional area A, is supported
+    as shown below and loaded on the overhangs by a uniformly distributed
+    load w. Determine the maximum bending stress in the middle portion of
+    the beam and the deflection :math:`\delta` at the middle of the beam.
+
+Reference:
+   -  S. Timoshenko, Strength of Material, Part I, Elementary Theory and
+   Problems, 3rd Edition, D. Van Nostrand Co., Inc., New York, NY, 1955,
+   pg. 98, problem 4.
+
+Analysis Type(s):
+ - Static Analysis ``ANTYPE=0``
+
+Element Type(s):
+ - 3-D 2 Node Beam (BEAM188)
+
+.. image:: ../../_static/vm2_setup.png
+   :width: 400
+   :alt: VM2 Problem Sketch
+
+Material Properties
+ - :math:`E = 30 \cdot 10^6 psi`
+
+Geometric Properties:
+ - :math:`a = 120 in`
+ - :math:`l = 240 in`
+ - :math:`h = 30 in`
+ - :math:`A = 50.65 in^2`
+ - :math:`I_z = 7892 in^4`
+
+Loading:
+ - :math:`w = (10000/12) lb/in`
+
+Analytical Equations:
+ - :math:`P = R_1 + R_2` where :math:`P` is load.
+ - :math:`\frac{R_2}{R_1} = \frac{a}{b}`
+   Where :math:`a` and :math:`b` are the ratios of distances between
+   the load and the wall.
+
+"""
+
+
+# sphinx_gallery_thumbnail_path = '_static/vm2_setup.png'
+
+###############################################################################
+# Start MAPDL
+# ~~~~~~~~~~~
+
+from ansys.mapdl.core import launch_mapdl
+
+# start mapdl and clear it
+mapdl = launch_mapdl()
+mapdl.clear()  # optional as MAPDL just started
+
+# enter verification example mode and the pre-processing routine
+mapdl.verify()
+mapdl.prep7()
+
+###############################################################################
+# Define Element Type
+# ~~~~~~~~~~~~~~~~~~~
+# Set up the element type (a beam-type).
+
+mapdl.antype("STATIC")
+mapdl.et(1, "BEAM188")
+
+
+###############################################################################
+# Define Material
+# ~~~~~~~~~~~~~~~
+# Set up the material and its type (a single material, with a beam-type section).
+
+mapdl.mp("EX", 1, 30E6)
+mapdl.mp("PRXY", "", 0.3)
+
+###############################################################################
+# Define Section
+# ~~~~~~~~~~~~~~
+# Set up the cross-section for a beam element.
+
+mapdl.run("SECT,1,BEAM,I")
+mapdl.run("W_F=1.048394965")
+mapdl.run("W_W=0.6856481")
```

Comment:
These should be python variables.  Let's avoid using MAPDL variables.

```suggestion
w_f = 1.048394965
w_w = 0.6856481
```